save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/ExecutionStatistics.kt

```diff
@@ -21,11 +26,84 @@ external interface ExecutionStatisticsProps : Props {
      * And instance of [ExecutionDto], which should be passed from parent component
      */
     var executionDto: ExecutionDto?
+}
+
+/**
+ * Class contains all execution statistics values for rending
+ *
+ * @param executionDto
+ */
+internal class ExecutionStatisticsValues(executionDto: ExecutionDto?) {
```

Comment:
I think we'll need (maybe, in a separate PR) to move this class to save-cloud-common, because we'll need statistics for all REST API client too

Reply:
it was done only for testing purposes (to test calculation of rates and state).